styles/components/_error.scss

```diff
@@ -0,0 +1,28 @@
+body{
+    height: 100vh;
+    background-color: $body-background-color;
+}
+
+main{
+    .error-message{
+        padding-top: 288px; //384 - 96
```

Comment:
Avoid directly setting a fixed padding-top value like 288px for layout adjustments. This approach can cause issues with responsiveness and maintainability, especially on different screen sizes. Instead, it’s better to use a more flexible approach, such as layout positioning or calculations based on parent dimensions.

In our next meeting, I’ll share an improved solution to achieve the desired layout using a more responsive and modern approach. We'll go through it together to ensure everything is clear!